CMakeLists.txt

```diff
@@ -22,7 +22,9 @@ endif()
 
 if(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++latest")
```

Comment:
minor nit pick, but you can throw them all on the same line.

set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++latest /MP /FS")